%YAML:1.0

#common parameters
savePath: "/home/cxn/myfile/OR_ws/orchardrover_ws/src/OrchardRover/or_lasercamcal/log/"
bag_path: "/home/cxn/myfile/OR_ws/orchardrover_ws/2021-04-29-22-22-10.bag"

scan_topic_name: "/scan"
img_topic_name: "/usb_cam/image_raw"

# tag info
tag_type: 1 # 1: kalibr_tag, 2: apriltag
tag_size: 0.055 # tag size, unit meter
tag_spacing: 0.3 # tag spacing, only used in kalibr_tag. For details, please see kalibr tag description.
black_border: 2 # if you use kalibr_tag black_boarder = 2; if you use apriltag black_boarder = 1

#camera calibration 
model_type: PINHOLE
camera_name: camera
image_width: 720
image_height: 540

distortion_parameters:
   k1: -0.3827
   k2: 0.1633
   p1: 0.0
   p2: 0.0

projection_parameters:
   fx: 525.3219
   fy: 525.4360
   cx: 363.4922
   cy: 278.2270

